rails/config/routes.rb

```diff
@@ -153,7 +188,7 @@
     match '/signup' => 'users#new', :as => :signup, via: [:get, :post]
     match '/activate/:activation_code' => 'users#activate', :as => :activate, :activation_code => nil, via: [:get, :post]
     match '/forgot_password' => 'passwords#login', :as => :forgot_password, via: [:get, :post]
-    match '/forgot_password/email' => 'passwords#login', :as => :forgot_password_email, via: [:get, :post]
+    match '/forgot_password/email' => 'passwords#email', :as => :forgot_password_email, via: [:get, :post]
```

Comment:
I'm a little worried about this. 
@emcelroy perhaps you are using `/forgot_passdword/email` in some collection pages?

There is no view that goes with `passwords#email`, so I'm not sure what rails will render if someone hits this updated route. Witht he old route it would render the login view which calls `PortalComponents.renderForgotPasswordModal({})`

Reply:
I'll leave this be for now.

Reply:
I've reverted this back to `#passwords/login`